getting-started/gopher-run/cmd/training.sh

```diff
@@ -31,23 +31,26 @@ MODEL_NAME="${DATASET_NAME}_${ALGORITHM}_${MODEL_TYPE}"
 JOB_DIR="gs://${BUCKET_NAME}/algorithms_training/${MODEL_NAME}/"
 FRAMEWORK="XGBOOST"
 
-DATE="$(date '+%Y%m%d_%H%M%S')"
+while true; do
+  DATE="$(date '+%Y%m%d_%H%M%S')"
 
-# JOB_ID="${MODEL_NAME}_${DATE}"
-# gcloud beta ai-platform jobs submit training $JOB_ID \
-# --master-image-uri=$IMAGE_URI --scale-tier=BASIC --job-dir=$JOB_DIR \
-# -- \
-# --preprocess --training_data_path=$TRAINING_DATA_PATH --objective=multi:softmax --num_class=4
-    
-VERSION_NAME="V_${DATE}"
-gcloud ai-platform versions create $VERSION_NAME \
---model $MODEL_NAME \
---origin "${JOB_DIR}model/" \
---runtime-version=1.11 \
---framework $FRAMEWORK \
---python-version=2.7
+  JOB_ID="${MODEL_NAME}_${DATE}"
+  gcloud beta ai-platform jobs submit training $JOB_ID \
+  --master-image-uri=$IMAGE_URI --scale-tier=BASIC --job-dir=$JOB_DIR \
+  -- \
+  --preprocess --training_data_path=$TRAINING_DATA_PATH --objective=multi:softmax --num_class=4
+      
+  VERSION_NAME="V_${DATE}"
+  gcloud ai-platform versions create $VERSION_NAME \
+  --model ${MODEL_NAME} \
+  --origin "${JOB_DIR}model/" \
+  --runtime-version=1.11 \
+  --framework ${FRAMEWORK} \
+  --python-version=2.7
+  gcloud ai-platform versions set-default ${VERSION_NAME} --model=${MODEL_NAME}
 
-gcloud ai-platform versions set-default ${VERSION_NAME} --model=${MODEL_NAME}
+  sleep 1800
```

Comment:
Does this need to be running/looping somewhere? If so, I'll run it in case you don't have remote access. Is 30 minutes enough for it to finish?

Reply:
I started it on my machine